dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/blob.properties

```diff
@@ -12,6 +12,29 @@ implementation=objectstorage
 # Optional, defaults to 32768 bytes (32KB), must be positive
 hybrid.size.threshold=32768
 
+# ========================================= Cassandra BlobStore Cache ======================================
+# A cassandra cache can be enabled to reduce latency when reading small blobs frequently
+# A dedicated keyspace with a replication factor of one is then used
+# Cached eviction policy is TTL based
+# Only blobs below a given threshold will be stored.
```

Comment:
Did we do `<=` threadHoldSize, `below` should be `with in`

Reply:
IMO `within` should be used with a range, and a range cannot be represented as a upper bound, even if it is inclusive.

I also don't think that the upper bound being inclusive makes much of a difference, so I rather keep it as is.